chaincfg/init.go

```diff
@@ -60,7 +60,7 @@ func shift(mask uint16) uint {
 
 func validateChoices(mask uint16, choices []Choice) error {
 	var (
-		isIgnore, isNo int
+		isAbstain, isNo int
```

Comment:
They're counts (obviously poorly named ones).  I just went through and did a find/replace.